docs/content/node-operation/index.mdx

```diff
@@ -1,21 +1,42 @@
 ---
-title: Hello Node Operator!
+title: Flow Node
```

Comment:
I think it might be better to always refer to `Nodes` - in plural. We have different types of nodes and we wanted to stay on the network-level that consists of multiple nodes

Reply:
Alright, noted.